### PR TITLE
working on ticket 6

### PR DESCRIPTION
**Pull request by kazeemerinfolami**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -132,3 +132,46 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET: 200 response with an object object containing all the comments if article_id exist", () => {
```
Comment (JVal-NC) [inline __tests__/app.test.js:137]: 'object object'

File: controller/nc_news.controller.js [hunk 2]
```diff
@@ -47,3 +47,5 @@ exports.getCommentsByArticle_id = (req, res, next) => {
       next(err);
     });
 };
+
+// exports.post = ()
```
Comment (JVal-NC) [inline controller/nc_news.controller.js:51]: Please remove commented out code

File: __tests__/app.test.js [hunk 3]
```diff
@@ -143,6 +143,8 @@ describe("/api/articles/:article_id/comments", () => {
         const { comments } = res.body;
         //if topic is an array
         expect(comments).toBeInstanceOf(Array);
+        //comments should be served with the most recent comments first
```
Comment (JVal-NC) [inline __tests__/app.test.js:146]: article_id value of 1 could be expected in this test

Comment (JVal-NC) [inline __tests__/app.test.js:146]: Extra test needed for comments to be an empty array for valid review_id with no comments

**Review by JVal-NC — COMMENTED** [review]
Ticket 6 review comments
